chat.go

```diff
@@ -51,7 +56,31 @@ type PromptAnnotation struct {
 	ContentFilterResults ContentFilterResults `json:"content_filter_results,omitempty"`
 }
 
+type ChatMessageImageURL struct {
+	URL string `json:"url"`
```

Comment:
You can also add detail attribute. https://platform.openai.com/docs/guides/vision/calculating-costs

Reply:
I made it like this
```go
type ImageURLDetail string

const (
	ImageURLDetailLow  ImageURLDetail = "low"
	ImageURLDetailHigh ImageURLDetail = "high"
)

type ChatMessageImageURL struct {
	Detail ImageURLDetail `json:"detail"`
	URL    string         `json:"url"`
}
```